NOTICE.md

```diff
@@ -0,0 +1,65 @@
+# Crystal Programmig Language
+
+Copyright 2012-2017 Manas Technology Solutions.
+
+This product includes software developed at Manas Technology Solutions (<https://manas.tech/>).
+
+Apache License v2.0 applies to all works.
+
+Please see [LICENSE](/LICENSE) for additional copyright and licensing information.
+
+## External libraries information
+
+Crystal compiler links these libraries, which have their own license:
+
+  * [LLVM](http://llvm.org/)
+    - [BSD-3, effectively](http://releases.llvm.org/2.8/LICENSE.TXT)
+  * [PCRE](http://pcre.org/)
+    - [BSD-3](https://opensource.org/licenses/BSD-3-Clause)
+  * [libevent2](http://libevent.org/)
+    - BSD-3
+  * [libiconv](https://www.gnu.org/software/libiconv/)
+    - [LGPLv3](https://www.gnu.org/licenses/lgpl-3.0.en.html)
+  * [bdwgc](http://www.hboehm.info/gc/)
+    - [MIT](https://opensource.org/licenses/MIT)
+  * [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/)
+    - [GPLv3](https://www.gnu.org/licenses/gpl-3.0.en.html)
+
+Crystal standard library uses these libraries, which have their own licenses:
```

Comment:
No. "*Parts of the Crystal standard library*" sounds good, but "*link*" is really no thanks. Because, if Crystal standard library **links** to the GPL libraries in fact, we must license it GPL. Howeber, Crystal standard library work and exist independent of some GPL library (readline and GMP). I prefer "use" to "link".